lesson_9.py

```diff
@@ -0,0 +1,83 @@
+import threading
+import os
+
+
+def set_interval(func, sec):
```

Comment:
What is this for :smile: ? Can you include it in the class, as a method?

Reply:
I had some fun making the vehicle driveable until it reaches top speed and updates the miles driven in that time, if you run the code in pycharm you can see :).
It took me at least double the time to make this "extra" thing work compared to the actual homework, the two imports are directly connected to this exercise.